app/config/index.js

```diff
@@ -50,7 +51,8 @@ const config = {
     useConnectionString: process.env.AZURE_STORAGE_USE_CONNECTION_STRING,
     storageAccount: process.env.AZURE_STORAGE_ACCOUNT_NAME
   },
-  complianceApplicationCount: process.env.COMPLIANCE_APPLICATION_COUNT
+  complianceApplicationCount: process.env.COMPLIANCE_APPLICATION_COUNT,
+  complianceScheduler: process.env.COMPLIANCE_SCHEDULER
```

Comment:
Here I'd probably create a new compliance "section" similar to "storage"

<img width="584" alt="image" src="https://user-images.githubusercontent.com/118533775/205033116-db83c428-b195-4920-b39d-c628f318021a.png">